src/commandline.h

```diff
@@ -0,0 +1,151 @@
+#pragma once
```

Comment:
Single comment for `commandline.h` and `commandline.cpp`:
- There is a mix of `std::string` and `std::wstring` in there, would this be a problem, in particular if dealing with non-ascii characters in paths?
- I think a few comments there and there would be helpful... At least on the declarations.

Reply:
Non-ascii characters are apparently broken, thought I had checked. Nice catch, I'll fix those in the next PR.